.github/workflows/compile.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: [ubuntu-18.04]
+    steps:
+    - name: Install opam2
+      run: |
+        sudo add-apt-repository -y ppa:avsm/ppa
```

Comment:
Could also download `https://github.com/ocaml/opam/releases/download/2.0.8/opam-2.0.8-x86_64-linux` instead of using the ppa?

Reply:
Probably makes sense to check that it builds on both.